### DISCUSSION TITLE
Apache crashes on RPi4 with mod_auth_openidc enabled

### DISCUSSION THREAD

**Discussion by Ra72xx** [OpenIDC/mod_auth_openidc#835] [General]
I'm not really an expert for apache configurations, so maybe I'm missing something here:
I have a fairly standard RPi4 with 32bit Raspbian OS (Debian 11 based) here and try to use this module (noteworthy changes to default apache config are usage of mpm_event and http2, and auth_ldap being activated). 

However, apache2 simple crashes without any hint when enabling the module. The log shows me
```
Apr 08 13:59:07 wolke systemd[1]: Starting The Apache HTTP Server...
Apr 08 13:59:07 wolke systemd[1]: Started The Apache HTTP Server.
Apr 08 13:59:07 wolke systemd[1]: apache2.service: Main process exited, code=killed, status=7/BUS
Apr 08 13:59:07 wolke systemd[1]: apache2.service: Failed with result 'signal'.
```
I'm at a loss where to start looking for a solution....

**Comment by zandbelt** [comment]
which version are you using and where did you get it from?

**Reply by Ra72xx** [reply]
Apache2 as provided by Raspbian:

```
root@pi:/home/pi# apachectl -V
Server version: Apache/2.4.53 (Raspbian)
Server built:   2022-03-14T16:28:35
Server's Module Magic Number: 
Server loaded:  APR 1.7.0, APR-UTIL 1.6.1, PCRE 8.44 2020-02-12
Compiled using: APR 1.7.0, APR-UTIL 1.6.1, PCRE 8.39 2016-06-14
Architecture:   32-bit
Server MPM:     event
  threaded:     yes (fixed thread count)
    forked:     yes (variable process count)
Server compiled with....
 -D APR_HAS_SENDFILE
 -D APR_HAS_MMAP
 -D APR_HAVE_IPV6 (IPv4-mapped addresses enabled)
 -D APR_USE_SYSVSEM_SERIALIZE
 -D APR_USE_PTHREAD_SERIALIZE
 -D SINGLE_LISTEN_UNSERIALIZED_ACCEPT
 -D APR_HAS_OTHER_CHILD
 -D AP_HAVE_RELIABLE_PIPED_LOGS
 -D DYNAMIC_MODULE_LIMIT=256
 -D HTTPD_ROOT="/etc/apache2"
 -D SUEXEC_BIN="/usr/lib/apache2/suexec"
 -D DEFAULT_PIDLOG="/var/run/apache2.pid"
 -D DEFAULT_SCOREBOARD="logs/apache_runtime_status"
 -D DEFAULT_ERRORLOG="logs/error_log"
 -D AP_TYPES_CONFIG_FILE="mime.types"
 -D SERVER_CONFIG_FILE="apache2.conf
```
For mod_openidc I tried both the version from Raspbian repo (2.4.9-1) and some more current version from Debian backports (I think) 2.4.11.1-1.

**Reply by zandbelt** [reply]
I'm sorry but I have no clue; there's only https://groups.google.com/g/mod_auth_openidc/c/4u7cI2uXFIw?pli=1

**Reply by Ra72xx** [reply]
I found that report also. In that case the problem seems to have been a wrong configuration. Is it possible that wrong trailing slashes or something like that crash apache2? Usually, this leads at least to a more meaningfull error message?!
I was just wondering that I'm the ony one with the problem as my config seems pretty straightforward and widely used.
Anyway, thanks for the really quick response!
